message_ix_models/tests/model/transport/test_report.py

```diff
@@ -119,7 +125,8 @@ def test_simulated_solution(request, test_context, regions="R12", years="B"):
     assert 0 < len(result)
 
 
-@mark.usefixtures("quiet_genno")
+@build.get_computer.minimum_version
+@mark.usefixtures("quiet_genno", "preserve_report_callbacks")
```

Comment:
Ah, this might resolve the issues we saw on message_data, right? 
BTW: once this migration is complete, you wanted to clean up on message_data/dev, IIRC. This might be more tedious than for other modules, which can simply drop their development branches. Please let me know if I can be of help :)

Reply:
Thank you, I will!